onnx/defs/math/defs.cc

```diff
@@ -2509,9 +2512,10 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Input(
             2,
             "a_zero_point",
-            "Zero point tensor for input 'A'. It's optional and default value is 0. It could be a scalar or a 1-D tensor, "
-            "which means a per-tensor or per-row quantization. If it's a 1-D tensor, its number of elements "
-            "should be equal to the number of rows of input 'A'.",
+            "Zero point tensor for input 'A'. It's optional and default value is 0. It could be a scalar or N-D tensor. "
+            "Scalar refers to per tensor quantization whereas N-D refers or per row quaization. "
+            "If the input is 2D of shape [M, K] then zero point tensor may be an M element vector [zp_1, zp_2, ..., zp_M]. "
+            "If the input is N-D tensor with shape [D1, D2, M, K] then zero point tesnor may have shape [D1, D2, M, 1]. ",
```

Comment:
But can it be [D1, D2, 1]? Or, do we allow only the two extremes of a zero-point for the entire tensor or a zero-point for each row?

Reply:
Or [D1, D2, 1, 1], I mean.

Reply:
right now we only allow per tensor or per row\col  zero points